### DISCUSSION TITLE
Lithuanian version of Stellarium

### DISCUSSION THREAD

**Discussion by taurasg1** [Stellarium/stellarium#3609] [Translations]
Hi, 
Could you bring back lithuanian translation of Stellarium. As seen in translations list there is 38% translations done , and the most important parts are already translated. i also added some translations.

**Comment by taurasg1** [comment]
I think it is hopeless to wait being it fully translated or even revised existing translations. There is not much "fools" like me to spent time on it. All i know in astronomy already translated (some hundred records). The most of stars, nebulaes etc does not have lithuanian names, or they are known by professionals only.

**Comment by alex-w** [comment]
Please check the fresh version (development snapshot) of Stellarium:
https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot

**Reply by taurasg1** [reply]
Thanks, Alex.
I have installed Stellarium v24 from the link provided (...assets (13), stellarium-24.0-81f3739-qt6-win64.exe)  ,but did not found Lithuanian language. Perhaps i  am doing something wrong?

**Reply by alex-w** [reply]
Did you selected the language?

![stellarium-002](https://github.com/Stellarium/stellarium/assets/88731/ed867356-98d1-403d-8744-86ec9425c78f)

**Reply by gzotti** [reply]
I can confirm Lithuanian (LT) is not in the git repo.

```
git log  -- po/stellarium/lt.po
commit f7b28c262bc33396b416154e5b1850fa3275758f
Author: Alexander V. Wolf <alex.v.wolf@gmail.com>
Date:   Sat Oct 8 23:47:11 2022 +0700

    Languages without translators was removed, sorry
```

Maybe languages with 20% (?) translated content should be re-added even without translators.

**Reply by alex-w** [reply]
Ouch... It's my fault

**Reply by taurasg1** [reply]
still no... cant print screen with languages list. Intalled  app added 2 hours ago
![image](https://github.com/Stellarium/stellarium/assets/157574371/f855918b-c2ba-4bd1-b449-93565169b027)

**Reply by alex-w** [reply]
OK, fixed now: https://github.com/Stellarium/stellarium/commit/e78b299762b70db9d261a7c7c6d534757933fa4d

The translations will be available for testing in next development snapshot, sorry for missing data.

**Reply by taurasg1** [reply]
Thank you,

Looking forward to be available to download it..... :)

**Reply by alex-w** [reply]
Please check the fresh version (development snapshot) of Stellarium:
https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot

**Reply by taurasg1** [reply]
Yesss !
Thank you , Alex.
This is what i dreamed some years.
There is some more translations not included here (like Pleyades constellation stars, made this week). But i am very happy. 
Will close this discussion.

**Comment by alex-w** [comment]
@taurasg1 I’ve give to you extra permissions and you can validate translations now
